### ISSUE TITLE
Get current firmware version

### ISSUE THREAD

**Issue by xiki808** [dresden-elektronik/gcfflasher#5]
Hello and thanks you for this! It would be great if GCFFlasher can give use more details about the adapter, such as the current firmware version running.

**Comment by manup** [comment]
Yes this will be added in future, I'd like to add some kind of command promt to execute various commands like query firmware version and store/restore configuration.

**Comment by mmattel** [comment]
I can only agree to this request, this is really not a big deal reading out the current firmware version and add it to the printout when using the list command. I am wondering why a request is needed for such a basic funtionality...

Sometimes it is better to do smaller steps which helps users a lot instead waiting long time for the big jump, just my 2c.

**Comment by tomtastic** [comment]
This is already implemented in the reset-device-function :
```
$ ./GCFFlasher_internal -r -d /dev/ttyACM0
GCFFlasher V3_17 (c) dresden elektronik ingenieurtechnik gmbh
Reboot device /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2287355-if00 (ConBee II)
deCONZ firmware version 26780700
```

**Comment by supertr0n** [comment]
The reset command does not output the firmware version for me (using GCFFlasher V4.10.0 on Debian Bookworm):

```
> sudo GCFFlasher -r -d /dev/ttyACM0
command UART reset done
command UART reset done
command UART reset done
command UART reset done
```
